### DISCUSSION TITLE
Is is possible to have absolute redirects for trailing slash?

### DISCUSSION THREAD

**Discussion by kryver** [vercel/next.js#24262] [Help]
Hi, we have an issue using next.js and redirects. We get this issue when we for example have a request to mydomain.com//anotherdomain.com (without trailing slash). We have trailingSlash true so this will cause a redirect to mydomain.com//anotherdomain.com/, but the Location header is //anotherdomain.se causing the anotherdomain.com to open.  Any custom redirect rules seems to be applied after any trailingSlash redirect so we can't manually override this. We believe it would be solved by having absolute path in the Location header, but there seems to be no option for this in next.

**Comment by akellbl4** [comment]
Hi,
I faced an issue like this and discovered that Next.js doesn't strip double slash. We managed it with stripping on the reverse proxy server in front of Next.js. It's is nginx in our case. But you can use any server that you wish.
